### ISSUE TITLE
Support for MediaPortal 1.4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#437]
```
Support for MediaPortal 1.4
http://forum.team-mediaportal.com/threads/my-anime-3.107475/page-49#post-994911


```

Original issue reported on code.google.com by `werndly...@gmail.com` on 19 May 2013 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
changes to GUIButton
http://forum.team-mediaportal.com/threads/my-anime-3.107475/page-50#post-995873
```

Original comment by `werndly...@gmail.com` on 21 May 2013 at 10:00

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.2.32
```

Original comment by `werndly...@gmail.com` on 28 May 2013 at 4:38
- Changed state: **Fixed**
